package.json

```
@@ -3,23 +3,17 @@
   "displayName": "Linear",
   "description": "Unofficial extension for managing your Linear projects from VS Code",
   "icon": "images/logo.png",
-  "version": "0.0.2",
+  "version": "0.0.5",
```

Currently [released](https://marketplace.visualstudio.com/items?itemName=strigo.linear) version is `0.0.3`
```suggestion
  "version": "0.0.4",
```